src/metabase/query_processor/card.clj

```diff
@@ -26,7 +26,7 @@
    [metabase.util.i18n :refer [tru]]
    [metabase.util.log :as log]
    [metabase.util.malli :as mu]
-   #_{:clj-kondo/ignore [:discouraged-namespace]}
+   ^{:clj-kondo/ignore [:discouraged-namespace]}
```

Comment:
cljfmt wants to sort these comments to the top of the `:require` list, but that breaks Kondo. The fix here is to change them from comments to metadata forms, which Kondo still understands correctly but cljfmt doesn't try to sort